### PR TITLE
First release PR

### PR DESCRIPTION
**Pull request by jhanca-robotecai**
Commit 603c294 was set as a release `v0.0.1` - this PR merges it to a newly created `main` branch.

### CODE REVIEW

File: src/rosbot_xl_whoami/setup.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import glob
+import os
+
+from setuptools import find_packages, setup
+
+package_name = "rosbot_xl_whoami"
+
+setup(
+    name=package_name,
+    version="0.0.0",
```
Comment (pijaro) [inline src/rosbot_xl_whoami/setup.py:10]: Set to a proper version number.

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[submodule "Project"]
+	path = Project
+	url = https://github.com/o3de/RobotVacuumSample.git
+        branch = development
```
Comment (pijaro) [inline .gitmodules:4]: Use tag or hash, the development branch might be too dangerous in the future.

Comment (jhanca-robotecai) [reply .gitmodules:4]: The release was already made and this PR only fixes the lack of a `main` branch. I believe the issues should be solved in the `development` branch and applied to a next release.

**Review by maciejmajek — APPROVED** [review]
LGTM

File: .gitmodules (hunk 2 above)
Comment (jhanca-robotecai) [reply .gitmodules:4]: This is referenced in #9

File: src/rosbot_xl_whoami/setup.py (hunk 1 above)
Comment (jhanca-robotecai) [reply src/rosbot_xl_whoami/setup.py:10]: I added the issue for that: #12

**Pull request merged**